core/src/main/java/novoda/lib/sqliteprovider/provider/SQLiteContentProviderImpl.java

```diff
@@ -96,14 +95,12 @@ private Uri insertSilently(Uri uri, ContentValues values) {
     protected int updateInTransaction(Uri uri, ContentValues values, String selection, String[] selectionArgs) {
         ContentValues insertValues = (values != null) ? new ContentValues(values) : new ContentValues();
```

Comment:
Actually, I think that by "null is a possible value" they mean that a null value in the map is possible, not that the `values` itself can be null.